modules/loadbalancer/manifests/maintenance.pp

```diff
@@ -4,6 +4,21 @@
 # into maintenance mode.
 #
 class loadbalancer::maintenance {
+  if ! defined(File['/etc/nginx/includes']) {
+    file { '/etc/nginx/includes':
+      ensure => 'directory',
+    }
+  }
+
+  if ! defined(File['/etc/nginx/includes/maintenance.conf']) {
```

Comment:
I'm not sure the `defined` function was used here in a conditional, and I'm not sure it's required.

Reply:
So that it doesn't recreate the file if it already exists? Does that mean the condition is no longer required?